src/usr/local/www/includes/functions.inc.php

```diff
@@ -177,7 +177,7 @@ function disk_usage($slice = '/') {
 }
 
 function swap_usage() {
-	exec("/usr/sbin/swapinfo", $swap_info);
+	exec("/usr/sbin/swapinfo | /usr/bin/grep Total", $swap_info);
```

Comment:
`swapinfo` does not output a line with `Total` for me, which makes this always empty/blank.

    : /usr/sbin/swapinfo
    Device          1K-blocks     Used    Avail Capacity
    /dev/label/swap0    524284        0   524284     0%